### PR TITLE
SkyTraq GPS query software version code overhaul. Fixes issue #8

### PR DESCRIPTION
**Pull request by tganley**
Also quality of life improvements.

### CODE REVIEW

File: hardware_interface/include/skytraq_binary.h [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#define GPS_UART_TIMEOUT_MS 1000 * portTICK_PERIOD_MS
+#define GPS_UART_TIMEOUT_MS 2000 * portTICK_PERIOD_MS
```
Comment (Taylorr82) [inline hardware_interface/include/skytraq_binary.h:8]: That's a very long time. Why so long?

File: equipment_handler/source/housekeeping_charon.c [hunk 2]
```diff
@@ -19,14 +19,18 @@
 
 #include "housekeeping_charon.h"
 
-int Charon_getHK(charon_housekeeping * hk){
+GPS_RETURNSTATE Charon_getHK(charon_housekeeping * hk){
     // Read temperature sensors
     if(readAllTemps(&hk->temparray[0])){
```
Comment (Taylorr82) [inline equipment_handler/source/housekeeping_charon.c:24]: What does this do exactly?

File: hardware_interface/include/skytraq_binary.h (hunk 1 above)
Comment (tganley) [reply hardware_interface/include/skytraq_binary.h:8]: There was one command that took like 1.5s

File: equipment_handler/source/housekeeping_charon.c (hunk 2 above)
Comment (tganley) [reply equipment_handler/source/housekeeping_charon.c:24]: It's a function for the housekeeping task to call that collects the temp data from I2C and the fimware CRC from the skytraq

Comment (Taylorr82) [reply equipment_handler/source/housekeeping_charon.c:24]: What I mean is what is the if statement checking for?

**Pull request merged**